src/agni/client/receiver/FileReceiver.java

```diff
@@ -1,5 +1,13 @@
-package agni.client.receiver;
+epackage agni.client.receiver;
 
 public class FileReceiver extends MessageParser {
 
+    public FileReceiver() {
+        super();
+        // TODO Auto-generated constructor stub
+    }
+
+    private void notifyFileReceived(String src, String file) {
```

Comment:
I think we should change the variable name to `filepath` instead of `file`. It should be more clear as to what the argument is actually supposed to be.